tests/test_revision.py

```diff
@@ -138,6 +138,17 @@ def test_bitbucket_history_link(self, revision_partial, enabled_context):
         assert "<a href=\"" + expected_page_source_url + "\">Bitbucket</a>" in rendered_revision
         assert_valid_html(rendered_revision)
 
+    def test_codeberg_history_link(self, revision_partial, enabled_context):
+        expected_page_source_url = "https://codeberg.org/myUsername/myRepository/commits/main/docs/index.md"
```

Comment:
another very small change request

can you rename this var?  i'm guessing you copied the bitbucket test but the codeberg use case is more similar to the github test
```suggestion
        expected_page_history_url = "https://codeberg.org/myUsername/myRepository/commits/main/docs/index.md"
```

if i remember correctly bitbucket didn't have a nice way to view file history on a standard URL so i made that config type link to the source page instead of the history page